tox.ini

```diff
@@ -0,0 +1,19 @@
+[tox]
+envlist = py36, py37, py38, py39
+skip_missing_interpreters = true
+
+[gh-actions]
+python =
+    3.6: py36
+    3.7: py37
+    3.8: py38
+    3.9: py39
+
+[testenv]
+deps = cython
+commands =
+    make
```

Comment:
In CI, we should (also?) check that the _committed_ `_readtags.c` file passes the tests. The `make` call regenerates that file and never commits the changes.

Reply:
`make` is taken from https://github.com/universal-ctags/python-ctags3/blob/master/.travis.yml. I would like to focus on migration. 
Fixing the issue in the original actions run in Travis can be fixed in another pull request.

Reply:
Oh, agreee.